assets/js/app.js

```diff
@@ -0,0 +1,6 @@
+"use strict"
+  const localization = document.getElementById("switchlang");
+  function handleClickLang() {
+    localization?.checked ? window.location.href = 'index_en.html': window.location.href = 'index.html'
```

Comment:
make your handler reusable. 
```suggestion
  function handleClickLang(ev) {
   // ev.something - research this object
   don't use localization?.checked defined in outer scope
```